pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +922,40 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAOperationsDuration tracks the duration of the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAOperationsDurationKey,
+			Help:           "Duration in seconds of the DRA operations (PrepareResources and UnprepareResources).",
+			Buckets:        metrics.DefBuckets,
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"operation_name"},
+	)
+
+	// DRAOperationsErrorsTotal is a counter that tracks the number of errors in the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsErrorsTotal = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAOperationsErrorsTotalKey,
+			Help:           "Cumulative number of errors when preparing and unpreparing resources.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"operation_name"},
+	)
+
+	DRAGRPCOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAGRPCOperationsDurationKey,
+			Help:           "Duration in seconds of the DRA gRPC operations",
+			Buckets:        metrics.DefBuckets,
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"driver_name", "method_name", "grpc_status_code"},
```

Comment:
I suggest to add boolean isError as a dimension.

Reply:
`grpc_status_code ` would be even better. I don't know if we should just follow the OpenTelemetry semantic convention. Maybe we can start with this metric? But I am not insisting here

Reply:
Let's start with `DRAOperationsDuration` and `DRAGRPCOperationsDuration` then.